tasks.robot

```diff
@@ -56,60 +60,18 @@ Collect Images From User
     ${confidence_threshold} =    Convert To Number    ${confidence_threshold}
     ${similarity_threshold} =    Convert To Number    ${similarity_threshold}
 
-    RETURN    ${result}[query_image]    ${result}[reference_image]
-    ...    ${confidence_threshold}    ${similarity_threshold}
-
-Ask For Retry
-    [Documentation]    Continue or exit the loop of signature verification process.
-
-    Add Submit Buttons    buttons=Retry,Exit    default=Exit
-    &{result} =    Run Dialog    title=${TITLE} - Results
-    ${retry} =    Run Keyword And Return Status    Should Be Equal As Strings
-    ...    ${result}[submit]    Retry
-    RETURN    ${retry}
-
-Display Similar Signatures
-    [Documentation]    Show similar signatures as image crops for manual inspection.
-    [Arguments]    ${qry_path}    ${qry_conf}    ${ref_path}    ${ref_conf}
-    ...    ${status}    ${similarity}
-
-    Clear Dialog
-
-    IF    ${status}
-        Add Icon    Success
-        Add Heading    Signatures match
-    ELSE
-        Add Icon    Warning
-        Add Heading    Signatures don't match
-    END
-
-    Add Heading    Similarity: ${similarity * 100}%
-    Add Text    The signature to check (confidence ${qry_conf * 100}%):
-    Add Image    ${qry_path}
-    Add Text    The trusted signature to compare with (confidence ${ref_conf * 100}%):
-    Add Image    ${ref_path}
```

Comment:
Same for query and reference confidence scores.

Reply:
Adding these back.

One question came into mind when thinking about this: What do the separate confidences actually mean? Does Base64 do an internal guess of what the signature "should look like" and the confidences tell what is the % change that they are copies of that signature? Or do they refer to some other confidence %?

Reply:
But they are displayed now

Reply:
Thanks!

Yes, is telling you how confident Base64AI is that's a signature and not some irrelevant junk. From docs of [Filter Matching Signatures](https://robocorp.com/docs/libraries/rpa-framework/rpa-documentai-base64ai/keywords#filter-matching-signatures):
<img width="706" alt="Screenshot 2023-02-22 at 18 25 43" src="https://user-images.githubusercontent.com/709053/220593229-30d8170b-92a1-4eee-b8fa-2f4aaad5b94c.png">